plugin/rewrite/rewrite_test.go

```diff
@@ -434,12 +438,32 @@ func optsEqual(a, b []dns.EDNS0) bool {
 	return true
 }
 
+type StaticProvider map[string]metadata.Func
```

Comment:
doesn't need to be exported.

why `static`?

Reply:
I reuse it in the rewrite_test
Static because the value provided is static. In fact it is better maybe to say Direct or Immediate or Constant

Reply:
hum.. you are right. Finally I was not able to reused it. I will revert to initial.